core/types/block.go

```diff
@@ -34,7 +34,7 @@ import (
 )
 
 var (
-	EmptyRootHash  = DeriveSha(Transactions{})
+	EmptyRootHash  = DeriveShaFromBMT(Transactions{})
```

Comment:
`EmptyRootHash` stands for empty patricia tree, and it is used in `ethclient`, and storage hash. So it should not be touched.

We use additional empty root, `EmptyBMTRootHash`, for empty binary merkle tree.